### PR TITLE
OSDOCS-3638: Release note for AWS/vSphere default compute node resources

### PR DESCRIPTION
**Pull request by mjpytlak**
Version(s):
CP to 4.11

Issue:
This PR addresses [OSDOCS-3638](https://issues.redhat.com/browse/OSDOCS-3638) and [CORS-2021](https://issues.redhat.com/browse/CORS-2021)

Link to docs preview:
[Default AWS and VMware vSphere compute node resources](http://file.rdu.redhat.com/mpytlak/osdocs-3638-rn/release_notes/ocp-4-11-release-notes.html#ocp-4-11-worker-nodes)

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -84,6 +84,10 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 ==== Support for Microsoft Hyper-V generation version 2
 By default, the installation program now deploys a Microsoft Azure cluster using Hyper-V generation version 2 virtual machines (VMs). If the installation program detects that the instance type selected for the VMs does not support version 2, it uses version 1 for the deployment.
 
+[id="ocp-4-11-worker-nodes"]
+==== Default AWS and VMware vSphere compute node resources
+Beginning with {product-title} 4.11, the installation program now deploys AWS and VMware vSphere compute nodes with an instance type that provides 4 vCPUs and 16 GB of virtual RAM.
```
Comment (rna-afk) [inline release_notes/ocp-4-11-release-notes.adoc:89]: In AWS we pick the instance type that provides 4 vCPUs and 16GB RAM but for Vsphere, we do not have instance types. We set the vCPUs and RAM ourselves.

Comment (mjpytlak) [reply release_notes/ocp-4-11-release-notes.adoc]: Thanks for the clarification. In light of that, I believe we do not have to say "how" the node is deployed, but rather simply that they are now deployed with 4 CPU/16 GB RAM.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -84,6 +84,10 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 ==== Support for Microsoft Hyper-V generation version 2
 By default, the installation program now deploys a Microsoft Azure cluster using Hyper-V generation version 2 virtual machines (VMs). If the installation program detects that the instance type selected for the VMs does not support version 2, it uses version 1 for the deployment.
 
+[id="ocp-4-11-worker-nodes"]
+==== Default AWS and VMware vSphere compute node resources
+Beginning with {product-title} 4.11, the installation program now deploys AWS and VMware vSphere compute nodes with 4 vCPUs and 16 GB of virtual RAM.
```
[the remark this answers is not in the document]
Comment (mjpytlak) [reply release_notes/ocp-4-11-release-notes.adoc]: Agreed. Good catch.

**Comment by mjpytlak** [comment]
@yunjiang29  @jinyunma Have a 4.11 release note for default AWS/vSphere compute node resources that is ready for QE review. PTAL. Thank you.

**Comment by jinyunma** [comment]
Installer PR https://github.com/openshift/installer/pull/5841 updates default cpu/memory on AWS/vSphere/GCP in 4.11, seems missing GCP here?

**Comment by mjpytlak** [comment]
> Installer PR [openshift/installer#5841](https://github.com/openshift/installer/pull/5841) updates default cpu/memory for compute node on AWS/vSphere/GCP in 4.11, seems missing GCP here?

GCP didn't land in 4.11.

**Comment by yunjiang29** [comment]
@mjpytlak lgtm, thanks for the updates!

**Comment by jinyunma** [comment]
LGTM

**Review by maxwelldb — APPROVED** [review]

**Pull request merged**